elasticsearch/prep-install.sh.origin

```diff
@@ -0,0 +1,6 @@
+ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch 
```

Comment:
I do not since I'm just consuming it via source.  Maybe a name change to not have an 'sh' extension?